### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by murtaja1**

### CODE REVIEW

**Review by Othmanosx — COMMENTED** [review]
LGTM

**Review by Othmanosx — APPROVED** [review]

**Review by ZhinAbubakr — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
 import React from 'react'
+import Container from './containers/Container'
 
 export default function App() {
 	return (
 		<div>			
-			
-
-			
+			<Container />
```
Comment (MuhannedNoman) [inline src/App.js]: Indentation is off! Also, is this coming from another PR!!

File: src/components/Dashboard/Dashboard.jsx [hunk 2]
```diff
@@ -1,163 +1,200 @@
...
+		})
+	}, [doubleFilter])
+	React.useEffect(() => {
+		const setResponsiveness = () => {
+			if (window.innerWidth < 812) {
+				setState((prevState) => ({ ...prevState, mobileView: true }))
+			}
+			if (window.innerWidth < 1210) {
+				setState((prevState) => ({ ...prevState, mediumView: true }))
+			}
```
Comment (MuhannedNoman) [inline src/components/Dashboard/Dashboard.jsx]: Why is responsiveness being handled like this!

File: src/components/Dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -1,163 +1,200 @@
...
-              <Post />
-            </div>
-          </Grid>
+			{mediumView && (
+				<Grid container justify="center" className={container}>
+					<Grid item xs={12} sm={8} lg={8}>
+						<div style={{ margin: "0 20px" }}>
+							<div className="messageSender">
+								<div className="messageSender_top">
+									<Avatar src={profile.imageUrl} />
```
Comment (MuhannedNoman) [inline src/components/Dashboard/Dashboard.jsx]: Indentation is off! Please insure prettier is working!!

File: src/components/Dashboard/Widgets/Cities.jsx [hunk 4]
```diff
@@ -1,72 +1,136 @@
-import React from "react";
+import React from "react"
 // import { makeStyles } from "@material-ui/core/styles";
```
Comment (MuhannedNoman) [inline src/components/Dashboard/Widgets/Cities.jsx:2]: Remove commented code!

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
Hey @murtaja1 thank you for this PR. I have noticed that this contains code from another PR which I have already reviewed so I didn't duplicate my comments! Also, I left you some comments take a look and let me know what you think!

File: src/components/Dashboard/Dashboard.jsx (hunk 2 above)
Comment (Othmanosx) [reply src/components/Dashboard/Dashboard.jsx]: I fixed this in my PR

**Pull request merged**